### PR TITLE
WEB-1099: checkout shipping design feedback

### PR DESCRIPTION
**Pull request by jnurse**
This PR addresses some of the design feedback for the checkout shipping page. 

 **JIRA**: https://mobify.atlassian.net/browse/WEB-1099
 **Linked PRs**: #292 

## Changes
- Updated email address label text
- Made h4s all caps
- Updated h4 spacing
- Added placeholder text for company and apt fields
- Set first shipping method option to selected by default

## How to test-drive this PR
- Run `npm run dev`
- Preview [Merlin's Potions](https://preview.mobify.com/?url=https%3A%2F%2Fwww.merlinspotions.com%2F&site_folder=https%3A%2F%2Flocalhost%3A8443%2Floader.js&disabled=0&domain=&scope=1)
- add an item to the cart
- navigate to: https://www.merlinspotions.com/checkout/
- confirm that the items checked off in the JIRA ticket have been fixed correctly

### CODE REVIEW

File: web/app/containers/checkout-shipping/partials/shipping-method.jsx [hunk 1]
```diff
@@ -25,16 +25,22 @@ class ShippingMethod extends React.Component {
...
                     {shippingMethods && shippingMethods.map(({label, cost, value}, idx) => {
                         return (
                             <FieldRow key={idx}>
-                                <ReduxForm.Field component={Field} name="shipping_method" label={<ShippingMethodLabel label={label} cost={cost} />}>
-                                    <input type="radio" noValidate value={value} />
+                                <ReduxForm.Field
+                                    component={Field}
+                                    name="shipping_method"
+                                    type="radio"
+                                    value={value}
```
Comment (mlworks) [inline web/app/containers/checkout-shipping/partials/shipping-method.jsx]: Question about redux form fields.  Any reason we moved the `value` away from the `<input />` to here?

Comment (jnurse) [inline web/app/containers/checkout-shipping/partials/shipping-method.jsx]: The value needs to be on the redux form field in order to get the radio button selected properly when it receives initialValues.

Comment (jnurse) [inline web/app/containers/checkout-shipping/partials/shipping-method.jsx]: I just double checked and the value prop does get added to the actual input in the markup.

Comment (mlworks) [inline web/app/containers/checkout-shipping/partials/shipping-method.jsx]: Ah good to know, thanks!

**Pull request merged**